core/api/views/cp_records.py

```diff
@@ -139,8 +139,10 @@ def _get_cp_history(self, cp_report, full_history=False):
             else cp_report
         )
         if cp_report_final:
-            history_qs = cp_report_final.cphistory.all().select_related(
-                "country_programme_report", "updated_by"
+            history_qs = (
+                cp_report_final.cphistory.all()
+                .select_related("country_programme_report", "updated_by")
+                .exclude(event_description__istartswith="status changed")
             )
             if not full_history:
                 history_qs = history_qs.filter(event_in_draft=False)
```

Comment:
your filter should be added here. If the full_history is set to true then we want to return all the history without any filtering